source/includes/fact-iso-date-objects.rst

```diff
@@ -1,5 +1,33 @@
-The argument can be any valid :ref:`expression
-<aggregation-expressions>` that resolves to a
-:doc:`BSON ISODate object </reference/bson-types>`,
-a :doc:`BSON Timestamp object </reference/bson-types>`,
-or a :doc:`Date object </reference/method/Date/>`.
+The argument must be a valid :ref:`expression
+<aggregation-expressions>` that resolves to one of the following:
+
+- A :ref:`Date <document-bson-type-date>` or
+  :ref:`Timestamp <document-bson-type-timestamp>`.
+
+- A document of the following form:
+
+  .. versionadded:: 3.6
```

Comment:
Oh,  the versionadded goes with just the timezone field.

So, you might want to pop it in the description of the timezone field.  Otherwise, it looks like the document you pass to X agg operator is totally new, when the date field was already there.